docs/fundamentals/code-analysis/quality-rules/ca1812.md

```diff
@@ -99,6 +94,16 @@ dotnet_diagnostic.CA1812.severity = none
 
 For more information, see [How to suppress code analysis warnings](../suppress-warnings.md).
 
+## Configure code to analyze
+
+Use the following option to configure which parts of your codebase to run this rule on.
+
+- [Ignore InternalsVisibleTo attribute](#ignore-internalsvisibleto-attribute)
+
+You can configure this option for just this rule, for all rules, or for all rules in this category (Performance). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
```

Comment:
@mavasani This makes it sound like the rule could be applicable to all rules/all rules in the category. If you set this rule to `false` for the category, does that work as a way to set it to `false` for both CA1812 and CA1852 with one config entry?